docs/antora/modules/ROOT/pages/setup/source.adoc

```diff
@@ -0,0 +1,342 @@
+There are several different methods and version managers used to install Node.js on your system.
+It is recommended to use one of the following two options:
+
+[tabs]
+====
+Option A - Node version manager::
++
+--
+It is recommended to use a Node version manager such as {url_nvm}[NVM^].
+NVM is a bash script that enables the management of multiple active Node.js versions.
+
+. Install NVM by following the {url_nvm_instructions}[official instructions^].
+. Install v16 of Node.js using NVM.
+
+.Check for the latest LTS version
+[source,bash]
+----
+nvm ls-remote
+----
+
+.Install the latest LTS version
+[source,bash]
+----
+nvm install 16
+----
+--
+Option B - Node.js package::
++
+--
+If NVM or other package managers are not required, it is possible to install the Node package globally  as shown in the following commands below:
+
+*Ubuntu*
+
+[source,bash]
+----
+curl -sL https://deb.nodesource.com/setup_16.x | sudo -E bash -
+sudo apt-get install -y nodejs
+----
+
+*macOS*
+
+For macOS, please execute the following command below:
+
+[source,bash]
+----
+brew install node@16
+----
+--
+====
+
+=== MySQL
+MySQL is used for storing persistent data.
+
+[NOTE]
+====
+Lisk Service, by default, expects a MySQL user `lisk` with the password `password` and a database `lisk`.
+
+**The MySQL user `lisk` needs to be manually created** after installing MySQL.
+
+You can choose to create a custom database with a different user and password. These values can be updated by setting the following environment variables (specific to each microservice): `SERVICE_INDEXER_MYSQL`, `SERVICE_APP_REGISTRY_MYSQL` and `SERVICE_STATISTICS_MYSQL` with the appropriate MySQL connection string. See the xref:{url_references_config}[]
+====
+
+[tabs]
+====
+Ubuntu::
++
+--
+[source,bash]
+----
+sudo apt update
+sudo apt install mysql-server=8.0*
+sudo mysql_secure_installation
+----
+
+--
+macOS::
++
+--
+[source,bash]
+----
+brew tap homebrew/services
+brew install mysql@8.0
+brew services start mysql@8.0
+----
+
+.Authentication
+
+If you encounter issues authenticating, regardless of which platform you are using, and you received the following error:
+
+ caching_sha2_password' cannot be loaded: dlopen(/usr/local/lib/plugin/caching_sha2_password.so, 2): image not found
+
+Try to perform the following: Change the `default_authentication_plugin` using the `mysql_native_password`.
+
+Open up `my.cnf` .
+
+If you are unsure where to find your `my.cnf`, run the following command:
+
+ $ mysql --verbose --help | grep my.cnf
+
+Add the following at the end of the file:
+
+ default_authentication_plugin=mysql_native_password
+
+Save and exit.
+
+Next, login via the terminal:
+
+ $ mysql -uroot
+
+Then run the following command to update the root password:
+
+ ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY '';
+
+Now you should be able to login to your MySQL 8 via your MySQL Client.
+
+
+--
+====
```

Comment:
If this is true for both platforms, please move the statement out of the tabs, so it is visible below no matter what tab is selected :)

```suggestion
--
====

.Authentication

If you encounter issues authenticating, and you received the following error:

 caching_sha2_password' cannot be loaded: dlopen(/usr/local/lib/plugin/caching_sha2_password.so, 2): image not found

Try to perform the following: Change the `default_authentication_plugin` using the `mysql_native_password`.

Open up `my.cnf` .

If you are unsure where to find your `my.cnf`, run the following command:

 $ mysql --verbose --help | grep my.cnf

Add the following at the end of the file:

 default_authentication_plugin=mysql_native_password

Save and exit.

Next, login via the terminal:

 $ mysql -uroot

Then run the following command to update the root password:

 ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY '';

Now you should be able to login to your MySQL 8 via your MySQL Client.

```

Reply:
Yes, good point Mona...Now done 👍